src/boundThreepids.js

```diff
@@ -0,0 +1,52 @@
+/*
```

Comment:
Maybe `BoundThreepids.js` with capital B would be more consistent?

Reply:
so nominally the ones with capitals are classes and the small-letter ones are collections of functions. This may not be the best naming convention / way of organising though.

Reply:
Hmm, I see. Should it be `bound-threepids.js` or `boundThreepids.sj` then? Seems unclear. Anyway, I guess doesn't need to be solved here.